detekt.yml

```
@@ -1,129 +1,129 @@
-    active: true
-
-coroutines:
-  GlobalCoroutineUsage:
-    active: true
-  RedundantSuspendModifier:
-    active: true
-  SleepInsteadOfDelay:
-    active: true
-  SuspendFunWithCoroutineScopeReceiver:
-    active: true
+# Root Detekt configuration — shared across all KalendarLib modules.
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)